builtin/providers/aws/resource_aws_autoscaling_scaling_policy_test.go

```diff
@@ -0,0 +1,125 @@
+package aws
+
+import(
+    "fmt"
+    //"math/rand"
+    //"strings"
+    "testing"
+    //"time"
+
+    "github.com/awslabs/aws-sdk-go/aws"
+    "github.com/awslabs/aws-sdk-go/service/autoscaling"
+    "github.com/hashicorp/terraform/helper/resource"
+    "github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSAutoscalingScalingPolicy_basic(t *testing.T) {
+    var policy autoscaling.ScalingPolicy
```

Comment:
`policy` doesn't seem to be used, other than assigning to it in `testAccCheckScalingPolicyExists`. Did you mean to check it's attributes later? If so, you can add a method that receives it like shown here in VPC tests:

- https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_vpc_test.go#L24-L25